source/Core/Src/settingsGUI.cpp

```diff
@@ -740,6 +744,16 @@ static void displayHallEffect(void) {
   }
 }
 static bool showHallEffect(void) { return getHallSensorFitted(); }
+static void displayHallEffectSleepTime(void) {
+  if (getSettingValue(SettingsOptions::HallEffectSleepTime)) {
+    OLED::printNumber(getSettingValue(SettingsOptions::HallEffectSleepTime) * 5, 2, FontStyle::LARGE, false);
+    OLED::print(LargeSymbolSeconds, FontStyle::LARGE);
+  } else {
+    // When sleep time is set to zero, we sleep for 1 second anyways. This is the default.
```

Comment:
Crazy idea, would it make more sense to have 0 disable the hall-effect and then the number default to like `1` or something?